package.json

```diff
@@ -34,6 +34,7 @@
   "homepage": "https://github.com/ChristopheBougere/asl-validator#readme",
   "dependencies": {
     "ajv": "^8.12.0",
+    "ajv-formats": "^3.0.1",
```

Comment:
added this for the `date-time` format of a Timestamp field

Reply:
And for jsonata as well, right?

Reply:
No. This lib provides format functions. The jsonata validation is done with a custom function that's simply a pass through to the jsonata library. The ajv-formats provides validators for date, date-time, uuid, and maybe some others. I'm only using the date-time one here. If we were already using `moment` or a similar lib, I could register a function to handle that validation w/o bringing in this dependency

Reply:
Ah ok, I got mixed up between `ajv.addFormat` and `addFormats` (from `ajv-formats`)